routers/routes/routes.go

```diff
@@ -466,6 +466,9 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/title", repo.UpdateIssueTitle)
 				m.Post("/content", repo.UpdateIssueContent)
 				m.Post("/watch", repo.IssueWatch)
+				m.Post("/add_time", repo.AddTimeManual)
```

Comment:
I also would propose to create a second group called `/time/stopwatch` for `/cancel` and `/toggle`.